provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/EnrollmentGroup.java

```diff
@@ -393,15 +396,18 @@ public void setAttestation(Attestation attestation)
         {
             throw new IllegalArgumentException("attestation cannot be null");
         }
-
-        /* SRS_ENROLLMENT_GROUP_21_040: [The setAttestation shall throw IllegalArgumentException if the attestation is not X509 signingCertificate.] */
-        if(!(attestation instanceof X509Attestation))
+        /* SRS_ENROLLMENT_GROUP_21_040: [The setAttestation shall throw IllegalArgumentException if the attestation is not X509 signingCertificate or SymmetricKeyAttestation] */
+        else if(!(attestation instanceof X509Attestation) && !(attestation instanceof SymmetricKeyAttestation))
         {
-            throw new IllegalArgumentException("attestation for EnrollmentGroup shall be X509");
+            throw new IllegalArgumentException("attestation for EnrollmentGroup shall be X509 or SymmetricKey");
         }
```

Comment:
Service will do this validation, so there's no need to add the same validation here.